prow/repoowners/repoowners.go

```diff
@@ -270,6 +270,9 @@ func (c *Client) LoadRepoOwners(org, repo, base string) (RepoOwner, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get current SHA for %s: %v", fullName, err)
 	}
+	if sha == "" {
+		return nil, fmt.Errorf("got an empty SHA for %s@heads/%s", fullName, base)
+	}
```

Comment:
The PR is of course fine, it makes the problem explicit and fail fast, but do we have any idea what is the underlying problem? When is it legit for `GetRef` to return an empty SHA?